MyApplication/app/src/main/java/net/serkanbal/wallmartlist/MainActivity.java

```diff
@@ -0,0 +1,118 @@
+package net.serkanbal.wallmartlist;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.Button;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import static net.serkanbal.wallmartlist.R.id.recyclerview;
+
+public class MainActivity extends AppCompatActivity {
+    public static final String GET_ITEMNAME = "http://api.walmartlabs.com/v1/search?format=json&apiKey=q8cpgj4hmeha6w47w4ab74e2&query=";
+    public List<ResultItem> mList = new ArrayList<>();
+    public Button mButtonPS4, mButtonXboxOne, mButtonWiiU;
+    public RecyclerView mRecyclerView;
+    public RecyclerView.Adapter mAdapter;
```

Comment:
Make all these `private` since you know you don't want any other classes to be able to modify them.